README.rst

```diff
@@ -66,6 +66,7 @@ provide your signature key to the URL builder.
 Domain Sharded URLs
 -------------------
 
+** Warning: Domain Sharding has been deprecated and will be removed in the next major release**
```

Comment:
I realize this is a bit terse but I was hoping to link to the blog post here when it's available. But maybe that can be a part of the next major release?

Reply:
I think as soon as we have documentation for this change available, we should post it early and often, in every relevant place.